conftest.py

```diff
@@ -80,17 +75,26 @@ def auto_libs(doctest_namespace):
 
 
 @pytest.fixture(autouse=True)
-def auto_libertem(doctest_namespace):
-    import libertem
-    import libertem.utils
-    import libertem.utils.generate
-    import libertem.masks
-    import libertem.api
+def auto_blobfinder(doctest_namespace):
     import libertem_blobfinder
-
-    doctest_namespace["libertem"] = libertem
-    doctest_namespace["libertem.utils"] = libertem.utils
-    doctest_namespace["libertem.utils.generate"] = libertem.utils.generate
-    doctest_namespace["libertem.masks"] = libertem.masks
-    doctest_namespace["libertem.api"] = libertem.api
     doctest_namespace["libertem_blobfinder"] = libertem_blobfinder
+
+
+@pytest.fixture(autouse=True)
+def auto_libertem(doctest_namespace):
+    try:
+        import libertem
+        import libertem.utils
+        import libertem.utils.generate
+        import libertem.masks
+        import libertem.api
+        import libertem_blobfinder
+
+        doctest_namespace["libertem"] = libertem
+        doctest_namespace["libertem.utils"] = libertem.utils
+        doctest_namespace["libertem.utils.generate"] = libertem.utils.generate
+        doctest_namespace["libertem.masks"] = libertem.masks
+        doctest_namespace["libertem.api"] = libertem.api
+        doctest_namespace["libertem_blobfinder"] = libertem_blobfinder
```

Comment:
Should these also use the `libertem_blobfinder` locations? Are these even still used in doctests?

If not:

```suggestion
        import libertem
        import libertem.api
        import libertem_blobfinder

        doctest_namespace["libertem"] = libertem
        doctest_namespace["libertem.api"] = libertem.api
        doctest_namespace["libertem_blobfinder"] = libertem_blobfinder
```